internal/web/applications_controller.go

```diff
@@ -0,0 +1,76 @@
+package web
+
+import (
+	"html/template"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/suse/carrier/internal/filesystem"
+)
+
+type ApplicationsController struct {
+}
+
+func (hc ApplicationsController) Index(w http.ResponseWriter, r *http.Request) {
+	Render([]string{"main_layout", "icons", "applications_index"}, w, r, map[string]interface{}{"serverUrl": r.Host})
```

Comment:
I cannot understand how r.Host information is used ?

Reply:
`serverUrl` is a left-over. I though `r.Host` would be what I would need to compare to the various link  `href`s but it turns out it was `r.Url`: https://github.com/SUSE/carrier/pull/235/files#diff-ecc48de753581438ba869ceff1df561c7fcf30b3baf38d6e19b20a658e11f6a8R73

I will remove it.